app/game/game.js

```diff
@@ -324,15 +325,29 @@ Game.prototype.calculateScores = function() {
   }, this);
 
   // Count VP Cards and Longest Rd, Biggest Army
-  this.players.forEach(function(player) {
+/*  var player_with_longest_road;
+  var player_road_length = 0;
+  for (player of this.players) {
+    var length = this.board.longest_road_for_player(player.id);
+    if (length > player_road_length)
+      player_with_longest_road = player.id;
+      player_road_length = length;
+  };
+*/
+  var player_with_longest_road = -1;
+  var player_road_length = 0;
+  for (var p=0; p<this.players.length; p++) {
+    if (this.players[p].score.longest_road > player_road_length) {
+      player_with_longest_road = this.players[p].id;
+      player_road_length = this.players[p].score.longest_road;
+    }
+  }
 
+  this.players.forEach(function(player) {
     player.score.victory_points = player.cards.count_victory_cards();
-
-    player.score.total_points += (player.score.longest_road) ? 2 : 0;
+    player.score.total_points += (player.id === player_with_longest_road) ? 2 : 0;
```

Comment:
PR looks great.  One change here.... road building only starts when 5 roads are joined.

Reply:
No problem, I'll bung that in and push as a new squash commit (this is forced, so it may nuke the branch if you have it checked out).